crates/sui-cluster-test/src/cluster.rs

```diff
@@ -178,13 +179,31 @@ impl Cluster for LocalNewCluster {
                 .expect("Unable to parse indexer address")
         });
 
-        let mut cluster_builder = TestClusterBuilder::new()
-            .set_genesis_config(genesis_config)
-            .enable_fullnode_events();
+        let mut cluster_builder = TestClusterBuilder::new().enable_fullnode_events();
 
-        if let Some(epoch_duration_ms) = options.epoch_duration_ms {
-            cluster_builder = cluster_builder.with_epoch_duration_ms(epoch_duration_ms);
+        // Check if we already have a config directory that is passed
+        if let Some(config_dir) = options.config_dir.clone() {
```

Comment:
When config_dir is provided, epoch_duration_ms will be ignored, we should error if they are both present